src/Boids.hs

```diff
@@ -26,6 +27,7 @@ import ViewPortTransform
 data World = World { worldBoids :: [Boid]
                    , worldGuide :: Point
                    , worldViewPort :: ViewPort
+                   , worldPrevPosition :: Maybe Point
```

Comment:
Rename this to something meaningful. When you look at this field out of the context it's not clear that it's a previous dragging position.